piwheels/master/the_scribe.py

```diff
@@ -52,6 +53,35 @@
 from .the_oracle import DbClient
 from .pypi import canonicalize_name
 
+UTC = timezone.utc
+
+
+ProjectRelease = namedtuple('ProjectRelease', (
+    'version', 'yanked', 'released', 'skip', 'stretch', 'buster', 'files',
+    'builds_succeeded', 'builds_failed'))
+
+def _pure_python_from_lower_abi(abi, files):
+    return any(
+        f['file_abi_tag'] == 'none' and abi > f['builder_abi']
```

Comment:
Is this meant to be abi greater-than or abi greater-than-or-equal-to? The latter would *seem* to be what we want but maybe I'm missing something.